docs/synapse/userguides/ug011_storm_basics.rst

```diff
@@ -117,11 +233,11 @@ Two ways to ask that question using Storm are:
 
 ``cli> ask #tc.t12 +inet:ipv4``
 
-The first query is problematic because it first asks Storm to return **all** ``inet:ipv4`` nodes within the hypergraph – potentially hundreds of thousands, or even millions of nodes, depending on how densely populated the hypergraph is (mathematically speaking, there are over four billion possible IPv4 addresses). Synapse has to lift **all** of those ``inet:ipv4`` nodes into memory and then select only those nodes with the ``tc.t12`` tag. The query is likely to take an extremely long time to return or to time out entirely (at least until query limits are incorporated into Synapse), and therefore represents a "bad" query.
+The first query is problematic because it first asks Storm to return **all** ``inet:ipv4`` nodes within the hypergraph – potentially hundreds of thousands, or even millions of nodes, depending on how densely populated the hypergraph is (mathematically speaking, there are over four billion possible IPv4 addresses). Synapse has to lift **all** of those ``inet:ipv4`` nodes into memory and then select only those nodes with the ``tc.t12`` tag. The query is likely to take an extremely long time to return (at least until query limits are incorporated into Synapse), and therefore represents a "bad" query.
 
 The second query first asks Storm to return **all** nodes tagged with ``tc.t12``. This may still be a large number depending on how much analysis and annotation has been performed related to Threat Group 12. However, the number of nodes tagged ``tc.t12`` will still be much smaller than the number of ``inet:ipv4`` nodes within a hypergraph. As such, the second query is more efficient or performant, and represents a "good" (or at least "better" query).
```

Comment:
Noted. I use "Threat Group 12" for a number of examples so I'll need to hunt-and-kill in some of the existing docs as well.